src/create_helpers.ts

```diff
@@ -592,9 +592,13 @@ export function createModule(doclet: INamespaceDoclet, nested: boolean, children
         flags |= ts.NodeFlags.NestedNamespace;
 
     if (children)
-        body = ts.createModuleBlock(children as ts.Statement[]);
+      body = ts.createModuleBlock(children as ts.Statement[]);
 
-    const name = ts.createStringLiteral(doclet.name);
+    let nameStr = doclet.name;
+    if (nameStr[0] === '"' && nameStr[nameStr.length - 1] === '"' || nameStr[0] === '\'' && nameStr[nameStr.length - 1] === '\''){
+      nameStr = nameStr.substr(1, nameStr.length - 2);
```

Comment:
This is not the same as the code I had suggested. It is missing some necessary parens and the spacing and brace format are both incorrect. I'd recommend using the suggestion I posted, or committing it directly from github.

Reply:
Updated per the suggestion. Changed suggested references of `name` to `nameStr` since name is defined later.